### ISSUE TITLE
Got ERROR while installing from pypi (for version 8.0)

### ISSUE THREAD

**Issue by JiaPai12138** [dashingsoft/pyarmor#949]
I got this error for version 8.0
```
ERROR: For req: pyarmor. Invalid script entry point: <ExportEntry pyarmor = pyarmor:None []> - A callable suffix is required. Cf https://packaging.python.org/specifications/entry-points/#use-for-scripts for more information.
```
I am using Python 3.10.10 (64 bit) on Windows 10 x64 (LTSC 21H2, OS build 19044.2604)

**Comment by JiaPai12138** [comment]
Looks like a simple workaround can fix this:
Modify setup.py, line 121:
```
    'pyarmor=pyarmor',
```
Back to:
```
    'pyarmor=pyarmor.pyarmor:main_entry',
```

Full result:
```
python setup.py install
running install
??????\Python310\lib\site-packages\setuptools\command\install.py:34: SetuptoolsDeprecationWarning: setup.py install is deprecated. Use build and pip and other standards-based tools.
  warnings.warn(
??????\Python310\lib\site-packages\setuptools\command\easy_install.py:144: EasyInstallDeprecationWarning: easy_install command is deprecated. Use build and pip and other standards-based tools.
  warnings.warn(
running bdist_egg
running egg_info
writing pyarmor.egg-info\PKG-INFO
writing dependency_links to pyarmor.egg-info\dependency_links.txt
writing entry points to pyarmor.egg-info\entry_points.txt
writing requirements to pyarmor.egg-info\requires.txt
writing top-level names to pyarmor.egg-info\top_level.txt
reading manifest file 'pyarmor.egg-info\SOURCES.txt'
reading manifest template 'MANIFEST.in'
adding license file 'LICENSE'
adding license file 'LICENSE-ZH'
adding license file 'LICENSE-ZH.7'
adding license file 'LICENSE.7'
writing manifest file 'pyarmor.egg-info\SOURCES.txt'
installing library code to build\bdist.win-amd64\egg
running install_lib
running build_py
creating build\bdist.win-amd64\egg
creating build\bdist.win-amd64\egg\pyarmor
copying build\lib\pyarmor\benchmark.py -> build\bdist.win-amd64\egg\pyarmor
copying build\lib\pyarmor\build_meta.py -> build\bdist.win-amd64\egg\pyarmor
creating build\bdist.win-amd64\egg\pyarmor\cli
copying build\lib\pyarmor\cli\config.py -> build\bdist.win-amd64\egg\pyarmor\cli
copying build\lib\pyarmor\cli\context.py -> build\bdist.win-amd64\egg\pyarmor\cli
copying build\lib\pyarmor\cli\default.cfg -> build\bdist.win-amd64\egg\pyarmor\cli
copying build\lib\pyarmor\cli\errors.py -> build\bdist.win-amd64\egg\pyarmor\cli
copying build\lib\pyarmor\cli\generate.py -> build\bdist.win-amd64\egg\pyarmor\cli
copying build\lib\pyarmor\cli\mixer.py -> build\bdist.win-amd64\egg\pyarmor\cli
copying build\lib\pyarmor\cli\register.py -> build\bdist.win-amd64\egg\pyarmor\cli
copying build\lib\pyarmor\cli\repack.py -> build\bdist.win-amd64\egg\pyarmor\cli
copying build\lib\pyarmor\cli\resource.py -> build\bdist.win-amd64\egg\pyarmor\cli
copying build\lib\pyarmor\cli\shell.py -> build\bdist.win-amd64\egg\pyarmor\cli
copying build\lib\pyarmor\cli\__init__.py -> build\bdist.win-amd64\egg\pyarmor\cli
copying build\lib\pyarmor\cli\__main__.py -> build\bdist.win-amd64\egg\pyarmor\cli
copying build\lib\pyarmor\cobuilder.py -> build\bdist.win-amd64\egg\pyarmor
copying build\lib\pyarmor\config.py -> build\bdist.win-amd64\egg\pyarmor
creating build\bdist.win-amd64\egg\pyarmor\examples
copying build\lib\pyarmor\examples\build-for-exe.bat -> build\bdist.win-amd64\egg\pyarmor\examples
copying build\lib\pyarmor\examples\build-for-freeze.bat -> build\bdist.win-amd64\egg\pyarmor\examples
copying build\lib\pyarmor\examples\build-with-project.bat -> build\bdist.win-amd64\egg\pyarmor\examples
copying build\lib\pyarmor\examples\build-with-project.sh -> build\bdist.win-amd64\egg\pyarmor\examples
creating build\bdist.win-amd64\egg\pyarmor\examples\cx_Freeze
copying build\lib\pyarmor\examples\cx_Freeze\hello.py -> build\bdist.win-amd64\egg\pyarmor\examples\cx_Freeze
copying build\lib\pyarmor\examples\cx_Freeze\queens.py -> build\bdist.win-amd64\egg\pyarmor\examples\cx_Freeze
copying build\lib\pyarmor\examples\cx_Freeze\setup.py -> build\bdist.win-amd64\egg\pyarmor\examples\cx_Freeze
creating build\bdist.win-amd64\egg\pyarmor\examples\helloworld
copying build\lib\pyarmor\examples\helloworld\foo.py -> build\bdist.win-amd64\egg\pyarmor\examples\helloworld
copying build\lib\pyarmor\examples\obfuscate-app.bat -> build\bdist.win-amd64\egg\pyarmor\examples
copying build\lib\pyarmor\examples\obfuscate-app.sh -> build\bdist.win-amd64\egg\pyarmor\examples
copying build\lib\pyarmor\examples\obfuscate-pkg.bat -> build\bdist.win-amd64\egg\pyarmor\examples
copying build\lib\pyarmor\examples\obfuscate-pkg.sh -> build\bdist.win-amd64\egg\pyarmor\examples
copying build\lib\pyarmor\examples\pack-obfuscated-scripts.bat -> build\bdist.win-amd64\egg\pyarmor\examples
copying build\lib\pyarmor\examples\pack-obfuscated-scripts.sh -> build\bdist.win-amd64\egg\pyarmor\examples
creating build\bdist.win-amd64\egg\pyarmor\examples\py2exe
copying build\lib\pyarmor\examples\py2exe\hello.py -> build\bdist.win-amd64\egg\pyarmor\examples\py2exe
copying build\lib\pyarmor\examples\py2exe\queens.py -> build\bdist.win-amd64\egg\pyarmor\examples\py2exe
copying build\lib\pyarmor\examples\py2exe\setup.py -> build\bdist.win-amd64\egg\pyarmor\examples\py2exe
creating build\bdist.win-amd64\egg\pyarmor\examples\pybench
copying build\lib\pyarmor\examples\pybench\Arithmetic.py -> build\bdist.win-amd64\egg\pyarmor\examples\pybench
copying build\lib\pyarmor\examples\pybench\Calls.py -> build\bdist.win-amd64\egg\pyarmor\examples\pybench
copying build\lib\pyarmor\examples\pybench\clockres.py -> build\bdist.win-amd64\egg\pyarmor\examples\pybench
copying build\lib\pyarmor\examples\pybench\CommandLine.py -> build\bdist.win-amd64\egg\pyarmor\examples\pybench
copying build\lib\pyarmor\examples\pybench\Constructs.py -> build\bdist.win-amd64\egg\pyarmor\examples\pybench
copying build\lib\pyarmor\examples\pybench\Dict.py -> build\bdist.win-amd64\egg\pyarmor\examples\pybench
copying build\lib\pyarmor\examples\pybench\Exceptions.py -> build\bdist.win-amd64\egg\pyarmor\examples\pybench
copying build\lib\pyarmor\examples\pybench\Imports.py -> build\bdist.win-amd64\egg\pyarmor\examples\pybench
copying build\lib\pyarmor\examples\pybench\Instances.py -> build\bdist.win-amd64\egg\pyarmor\examples\pybench
copying build\lib\pyarmor\examples\pybench\Lists.py -> build\bdist.win-amd64\egg\pyarmor\examples\pybench
copying build\lib\pyarmor\examples\pybench\Lookups.py -> build\bdist.win-amd64\egg\pyarmor\examples\pybench
copying build\lib\pyarmor\examples\pybench\NewInstances.py -> build\bdist.win-amd64\egg\pyarmor\examples\pybench
copying build\lib\pyarmor\examples\pybench\Numbers.py -> build\bdist.win-amd64\egg\pyarmor\examples\pybench
creating build\bdist.win-amd64\egg\pyarmor\examples\pybench\package
copying build\lib\pyarmor\examples\pybench\package\submodule.py -> build\bdist.win-amd64\egg\pyarmor\examples\pybench\package
copying build\lib\pyarmor\examples\pybench\package\__init__.py -> build\bdist.win-amd64\egg\pyarmor\examples\pybench\package
copying build\lib\pyarmor\examples\pybench\pybench.py -> build\bdist.win-amd64\egg\pyarmor\examples\pybench
copying build\lib\pyarmor\examples\pybench\Setup.py -> build\bdist.win-amd64\egg\pyarmor\examples\pybench
copying build\lib\pyarmor\examples\pybench\Strings.py -> build\bdist.win-amd64\egg\pyarmor\examples\pybench
copying build\lib\pyarmor\examples\pybench\systimes.py -> build\bdist.win-amd64\egg\pyarmor\examples\pybench
copying build\lib\pyarmor\examples\pybench\Tuples.py -> build\bdist.win-amd64\egg\pyarmor\examples\pybench
copying build\lib\pyarmor\examples\pybench\With.py -> build\bdist.win-amd64\egg\pyarmor\examples\pybench
copying build\lib\pyarmor\examples\README-ZH.md -> build\bdist.win-amd64\egg\pyarmor\examples
copying build\lib\pyarmor\examples\README.md -> build\bdist.win-amd64\egg\pyarmor\examples
creating build\bdist.win-amd64\egg\pyarmor\examples\simple
copying build\lib\pyarmor\examples\simple\queens.py -> build\bdist.win-amd64\egg\pyarmor\examples\simple
creating build\bdist.win-amd64\egg\pyarmor\examples\testmod
copying build\lib\pyarmor\examples\testmod\hello.py -> build\bdist.win-amd64\egg\pyarmor\examples\testmod
copying build\lib\pyarmor\examples\testmod\queens.py -> build\bdist.win-amd64\egg\pyarmor\examples\testmod
creating build\bdist.win-amd64\egg\pyarmor\examples\testpkg
copying build\lib\pyarmor\examples\testpkg\main.py -> build\bdist.win-amd64\egg\pyarmor\examples\testpkg
creating build\bdist.win-amd64\egg\pyarmor\examples\testpkg\mypkg
copying build\lib\pyarmor\examples\testpkg\mypkg\foo.py -> build\bdist.win-amd64\egg\pyarmor\examples\testpkg\mypkg
copying build\lib\pyarmor\examples\testpkg\mypkg\__init__.py -> build\bdist.win-amd64\egg\pyarmor\examples\testpkg\mypkg
creating build\bdist.win-amd64\egg\pyarmor\helper
copying build\lib\pyarmor\helper\buildext.py -> build\bdist.win-amd64\egg\pyarmor\helper
copying build\lib\pyarmor\helper\build_data_module.py -> build\bdist.win-amd64\egg\pyarmor\helper
copying build\lib\pyarmor\helper\get_bind_key.py -> build\bdist.win-amd64\egg\pyarmor\helper
copying build\lib\pyarmor\helper\get_license_info.py -> build\bdist.win-amd64\egg\pyarmor\helper
copying build\lib\pyarmor\helper\get_platform_name.py -> build\bdist.win-amd64\egg\pyarmor\helper
copying build\lib\pyarmor\helper\merge.py -> build\bdist.win-amd64\egg\pyarmor\helper
copying build\lib\pyarmor\helper\repack.py -> build\bdist.win-amd64\egg\pyarmor\helper
copying build\lib\pyarmor\helper\superuntime.py -> build\bdist.win-amd64\egg\pyarmor\helper
copying build\lib\pyarmor\helper\__init__.py -> build\bdist.win-amd64\egg\pyarmor\helper
copying build\lib\pyarmor\license.tri -> build\bdist.win-amd64\egg\pyarmor
copying build\lib\pyarmor\packer.py -> build\bdist.win-amd64\egg\pyarmor
creating build\bdist.win-amd64\egg\pyarmor\polyfills
copying build\lib\pyarmor\polyfills\argparse.py -> build\bdist.win-amd64\egg\pyarmor\polyfills
copying build\lib\pyarmor\polyfills\__init__.py -> build\bdist.win-amd64\egg\pyarmor\polyfills
copying build\lib\pyarmor\product.key -> build\bdist.win-amd64\egg\pyarmor
copying build\lib\pyarmor\project.py -> build\bdist.win-amd64\egg\pyarmor
copying build\lib\pyarmor\protect_code.pt -> build\bdist.win-amd64\egg\pyarmor
copying build\lib\pyarmor\protect_code2.pt -> build\bdist.win-amd64\egg\pyarmor
copying build\lib\pyarmor\public.key -> build\bdist.win-amd64\egg\pyarmor
copying build\lib\pyarmor\public_capsule.zip -> build\bdist.win-amd64\egg\pyarmor
copying build\lib\pyarmor\pyarmor-deprecated.py -> build\bdist.win-amd64\egg\pyarmor
copying build\lib\pyarmor\pyarmor-webui.py -> build\bdist.win-amd64\egg\pyarmor
copying build\lib\pyarmor\pyarmor.py -> build\bdist.win-amd64\egg\pyarmor
copying build\lib\pyarmor\pyimcore.py -> build\bdist.win-amd64\egg\pyarmor
copying build\lib\pyarmor\pyshield.key -> build\bdist.win-amd64\egg\pyarmor
copying build\lib\pyarmor\pyshield.lic -> build\bdist.win-amd64\egg\pyarmor
copying build\lib\pyarmor\pytransform.py -> build\bdist.win-amd64\egg\pyarmor
copying build\lib\pyarmor\README.rst -> build\bdist.win-amd64\egg\pyarmor
copying build\lib\pyarmor\register.py -> build\bdist.win-amd64\egg\pyarmor
copying build\lib\pyarmor\sppmode.py -> build\bdist.win-amd64\egg\pyarmor
copying build\lib\pyarmor\utils.py -> build\bdist.win-amd64\egg\pyarmor
copying build\lib\pyarmor\__init__.py -> build\bdist.win-amd64\egg\pyarmor
copying build\lib\pyarmor\__main__.py -> build\bdist.win-amd64\egg\pyarmor
byte-compiling build\bdist.win-amd64\egg\pyarmor\benchmark.py to benchmark.cpython-310.pyc
byte-compiling build\bdist.win-amd64\egg\pyarmor\build_meta.py to build_meta.cpython-310.pyc
byte-compiling build\bdist.win-amd64\egg\pyarmor\cli\config.py to config.cpython-310.pyc
byte-compiling build\bdist.win-amd64\egg\pyarmor\cli\context.py to context.cpython-310.pyc
byte-compiling build\bdist.win-amd64\egg\pyarmor\cli\errors.py to errors.cpython-310.pyc
byte-compiling build\bdist.win-amd64\egg\pyarmor\cli\generate.py to generate.cpython-310.pyc
byte-compiling build\bdist.win-amd64\egg\pyarmor\cli\mixer.py to mixer.cpython-310.pyc
byte-compiling build\bdist.win-amd64\egg\pyarmor\cli\register.py to register.cpython-310.pyc
byte-compiling build\bdist.win-amd64\egg\pyarmor\cli\repack.py to repack.cpython-310.pyc
byte-compiling build\bdist.win-amd64\egg\pyarmor\cli\resource.py to resource.cpython-310.pyc
byte-compiling build\bdist.win-amd64\egg\pyarmor\cli\shell.py to shell.cpython-310.pyc
byte-compiling build\bdist.win-amd64\egg\pyarmor\cli\__init__.py to __init__.cpython-310.pyc
byte-compiling build\bdist.win-amd64\egg\pyarmor\cli\__main__.py to __main__.cpython-310.pyc
byte-compiling build\bdist.win-amd64\egg\pyarmor\cobuilder.py to cobuilder.cpython-310.pyc
byte-compiling build\bdist.win-amd64\egg\pyarmor\config.py to config.cpython-310.pyc
byte-compiling build\bdist.win-amd64\egg\pyarmor\examples\cx_Freeze\hello.py to hello.cpython-310.pyc
byte-compiling build\bdist.win-amd64\egg\pyarmor\examples\cx_Freeze\queens.py to queens.cpython-310.pyc
byte-compiling build\bdist.win-amd64\egg\pyarmor\examples\cx_Freeze\setup.py to setup.cpython-310.pyc
byte-compiling build\bdist.win-amd64\egg\pyarmor\examples\helloworld\foo.py to foo.cpython-310.pyc
byte-compiling build\bdist.win-amd64\egg\pyarmor\examples\py2exe\hello.py to hello.cpython-310.pyc
byte-compiling build\bdist.win-amd64\egg\pyarmor\examples\py2exe\queens.py to queens.cpython-310.pyc
byte-compiling build\bdist.win-amd64\egg\pyarmor\examples\py2exe\setup.py to setup.cpython-310.pyc
byte-compiling build\bdist.win-amd64\egg\pyarmor\examples\pybench\Arithmetic.py to Arithmetic.cpython-310.pyc
byte-compiling build\bdist.win-amd64\egg\pyarmor\examples\pybench\Calls.py to Calls.cpython-310.pyc
byte-compiling build\bdist.win-amd64\egg\pyarmor\examples\pybench\clockres.py to clockres.cpython-310.pyc
byte-compiling build\bdist.win-amd64\egg\pyarmor\examples\pybench\CommandLine.py to CommandLine.cpython-310.pyc
byte-compiling build\bdist.win-amd64\egg\pyarmor\examples\pybench\Constructs.py to Constructs.cpython-310.pyc
byte-compiling build\bdist.win-amd64\egg\pyarmor\examples\pybench\Dict.py to Dict.cpython-310.pyc
byte-compiling build\bdist.win-amd64\egg\pyarmor\examples\pybench\Exceptions.py to Exceptions.cpython-310.pyc
byte-compiling build\bdist.win-amd64\egg\pyarmor\examples\pybench\Imports.py to Imports.cpython-310.pyc
byte-compiling build\bdist.win-amd64\egg\pyarmor\examples\pybench\Instances.py to Instances.cpython-310.pyc
byte-compiling build\bdist.win-amd64\egg\pyarmor\examples\pybench\Lists.py to Lists.cpython-310.pyc
byte-compiling build\bdist.win-amd64\egg\pyarmor\examples\pybench\Lookups.py to Lookups.cpython-310.pyc
byte-compiling build\bdist.win-amd64\egg\pyarmor\examples\pybench\NewInstances.py to NewInstances.cpython-310.pyc
byte-compiling build\bdist.win-amd64\egg\pyarmor\examples\pybench\Numbers.py to Numbers.cpython-310.pyc
byte-compiling build\bdist.win-amd64\egg\pyarmor\examples\pybench\package\submodule.py to submodule.cpython-310.pyc
byte-compiling build\bdist.win-amd64\egg\pyarmor\examples\pybench\package\__init__.py to __init__.cpython-310.pyc
byte-compiling build\bdist.win-amd64\egg\pyarmor\examples\pybench\pybench.py to pybench.cpython-310.pyc
byte-compiling build\bdist.win-amd64\egg\pyarmor\examples\pybench\Setup.py to Setup.cpython-310.pyc
byte-compiling build\bdist.win-amd64\egg\pyarmor\examples\pybench\Strings.py to Strings.cpython-310.pyc
byte-compiling build\bdist.win-amd64\egg\pyarmor\examples\pybench\systimes.py to systimes.cpython-310.pyc
byte-compiling build\bdist.win-amd64\egg\pyarmor\examples\pybench\Tuples.py to Tuples.cpython-310.pyc
byte-compiling build\bdist.win-amd64\egg\pyarmor\examples\pybench\With.py to With.cpython-310.pyc
byte-compiling build\bdist.win-amd64\egg\pyarmor\examples\simple\queens.py to queens.cpython-310.pyc
byte-compiling build\bdist.win-amd64\egg\pyarmor\examples\testmod\hello.py to hello.cpython-310.pyc
byte-compiling build\bdist.win-amd64\egg\pyarmor\examples\testmod\queens.py to queens.cpython-310.pyc
byte-compiling build\bdist.win-amd64\egg\pyarmor\examples\testpkg\main.py to main.cpython-310.pyc
byte-compiling build\bdist.win-amd64\egg\pyarmor\examples\testpkg\mypkg\foo.py to foo.cpython-310.pyc
byte-compiling build\bdist.win-amd64\egg\pyarmor\examples\testpkg\mypkg\__init__.py to __init__.cpython-310.pyc
byte-compiling build\bdist.win-amd64\egg\pyarmor\helper\buildext.py to buildext.cpython-310.pyc
byte-compiling build\bdist.win-amd64\egg\pyarmor\helper\build_data_module.py to build_data_module.cpython-310.pyc
byte-compiling build\bdist.win-amd64\egg\pyarmor\helper\get_bind_key.py to get_bind_key.cpython-310.pyc
byte-compiling build\bdist.win-amd64\egg\pyarmor\helper\get_license_info.py to get_license_info.cpython-310.pyc
byte-compiling build\bdist.win-amd64\egg\pyarmor\helper\get_platform_name.py to get_platform_name.cpython-310.pyc
byte-compiling build\bdist.win-amd64\egg\pyarmor\helper\merge.py to merge.cpython-310.pyc
byte-compiling build\bdist.win-amd64\egg\pyarmor\helper\repack.py to repack.cpython-310.pyc
byte-compiling build\bdist.win-amd64\egg\pyarmor\helper\superuntime.py to superuntime.cpython-310.pyc
byte-compiling build\bdist.win-amd64\egg\pyarmor\helper\__init__.py to __init__.cpython-310.pyc
byte-compiling build\bdist.win-amd64\egg\pyarmor\packer.py to packer.cpython-310.pyc
byte-compiling build\bdist.win-amd64\egg\pyarmor\polyfills\argparse.py to argparse.cpython-310.pyc
byte-compiling build\bdist.win-amd64\egg\pyarmor\polyfills\__init__.py to __init__.cpython-310.pyc
byte-compiling build\bdist.win-amd64\egg\pyarmor\project.py to project.cpython-310.pyc
byte-compiling build\bdist.win-amd64\egg\pyarmor\pyarmor-deprecated.py to pyarmor-deprecated.cpython-310.pyc
byte-compiling build\bdist.win-amd64\egg\pyarmor\pyarmor-webui.py to pyarmor-webui.cpython-310.pyc
byte-compiling build\bdist.win-amd64\egg\pyarmor\pyarmor.py to pyarmor.cpython-310.pyc
byte-compiling build\bdist.win-amd64\egg\pyarmor\pyimcore.py to pyimcore.cpython-310.pyc
byte-compiling build\bdist.win-amd64\egg\pyarmor\pytransform.py to pytransform.cpython-310.pyc
byte-compiling build\bdist.win-amd64\egg\pyarmor\register.py to register.cpython-310.pyc
byte-compiling build\bdist.win-amd64\egg\pyarmor\sppmode.py to sppmode.cpython-310.pyc
byte-compiling build\bdist.win-amd64\egg\pyarmor\utils.py to utils.cpython-310.pyc
byte-compiling build\bdist.win-amd64\egg\pyarmor\__init__.py to __init__.cpython-310.pyc
byte-compiling build\bdist.win-amd64\egg\pyarmor\__main__.py to __main__.cpython-310.pyc
creating build\bdist.win-amd64\egg\EGG-INFO
copying pyarmor.egg-info\PKG-INFO -> build\bdist.win-amd64\egg\EGG-INFO
copying pyarmor.egg-info\SOURCES.txt -> build\bdist.win-amd64\egg\EGG-INFO
copying pyarmor.egg-info\dependency_links.txt -> build\bdist.win-amd64\egg\EGG-INFO
copying pyarmor.egg-info\entry_points.txt -> build\bdist.win-amd64\egg\EGG-INFO
copying pyarmor.egg-info\requires.txt -> build\bdist.win-amd64\egg\EGG-INFO
copying pyarmor.egg-info\top_level.txt -> build\bdist.win-amd64\egg\EGG-INFO
zip_safe flag not set; analyzing archive contents...
pyarmor.__pycache__.__init__.cpython-310: module references __file__
pyarmor.__pycache__.benchmark.cpython-310: module references __file__
pyarmor.__pycache__.packer.cpython-310: module references __file__
pyarmor.__pycache__.pyarmor.cpython-310: module references __file__
pyarmor.__pycache__.pyimcore.cpython-310: module references __path__
pyarmor.__pycache__.pytransform.cpython-310: module references __file__
pyarmor.__pycache__.utils.cpython-310: module references __file__
pyarmor.cli.__pycache__.context.cpython-310: module references __file__
pyarmor.cli.__pycache__.resource.cpython-310: module references __file__
pyarmor.examples.testmod.__pycache__.hello.cpython-310: module references __file__
pyarmor.examples.testmod.__pycache__.hello.cpython-310: module MAY be using inspect.trace
pyarmor.helper.__pycache__.buildext.cpython-310: module references __file__
pyarmor.helper.__pycache__.get_license_info.cpython-310: module references __file__
pyarmor.helper.__pycache__.merge.cpython-310: module references __file__
creating 'dist\pyarmor-8.0.dev1-py3.10.egg' and adding 'build\bdist.win-amd64\egg' to it
removing 'build\bdist.win-amd64\egg' (and everything under it)
Processing pyarmor-8.0.dev1-py3.10.egg
creating ??????\python310\lib\site-packages\pyarmor-8.0.dev1-py3.10.egg
Extracting pyarmor-8.0.dev1-py3.10.egg to ??????\python310\lib\site-packages
Adding pyarmor 8.0.dev1 to easy-install.pth file
Installing pyarmor-script.py script to ??????\Python310\Scripts
Installing pyarmor.exe script to ??????\Python310\Scripts

Installed ??????\python310\lib\site-packages\pyarmor-8.0.dev1-py3.10.egg
Processing dependencies for pyarmor==8.0.dev1
Searching for pyarmor.cli.core==1.0
Best match: pyarmor.cli.core 1.0
Adding pyarmor.cli.core 1.0 to easy-install.pth file

Using ??????\python310\lib\site-packages
Finished processing dependencies for pyarmor==8.0.dev1
```
?????? is just where i install Python

**Comment by JiaPai12138** [comment]
[pyarmor-8.0-py2.py3-none-any.zip](https://github.com/dashingsoft/pyarmor/files/10917035/pyarmor-8.0-py2.py3-none-any.zip)
Change file extension name from ".zip" to ''.whl', install using pip
It works for me
```
Looking in indexes: https://pypi.org/simple, https://pypi.ngc.nvidia.com
Processing ??????\local\pyarmor-8.0-py2.py3-none-any.whl
Requirement already satisfied: pyarmor.cli.core~=1.0 in ??????\python310\lib\site-packages (from pyarmor==8.0) (1.0)
Installing collected packages: pyarmor
  Attempting uninstall: pyarmor
    Found existing installation: pyarmor 7.7.4
    Uninstalling pyarmor-7.7.4:
      Successfully uninstalled pyarmor-7.7.4
Successfully installed pyarmor-8.0
```

**Comment by JiaPai12138** [comment]
However, this workaround can install version 8.0 successfully, but has no 8.0 features

**Comment by vibhoraggarwal** [comment]
> [pyarmor-8.0-py2.py3-none-any.zip](https://github.com/dashingsoft/pyarmor/files/10917035/pyarmor-8.0-py2.py3-none-any.zip) Change file extension name from ".zip" to ''.whl', install using pip It works for me
> 
> ```
> pip install path\to\pyarmor-8.0-py2.py3-none-any.whl
> Looking in indexes: https://pypi.org/simple, https://pypi.ngc.nvidia.com
> Processing ??????\local\pyarmor-8.0-py2.py3-none-any.whl
> Requirement already satisfied: pyarmor.cli.core~=1.0 in ??????\python310\lib\site-packages (from pyarmor==8.0) (1.0)
> Installing collected packages: pyarmor
>   Attempting uninstall: pyarmor
>     Found existing installation: pyarmor 7.7.4
>     Uninstalling pyarmor-7.7.4:
>       Successfully uninstalled pyarmor-7.7.4
> Successfully installed pyarmor-8.0
> ```

In my setup it is installing from .why file and collects the required packages. However, it still throws the error (see image below). The version 7.1.0 works fine though.

![F61B7049-9B5E-4520-BCD4-1D9A0744C4D2](https://user-images.githubusercontent.com/19661841/224308996-d818ef39-db7f-4ab2-aa8c-560a8ee0c229.png)

**Comment by Blank-c** [comment]
```bash
~/.../test/test $ pip install pyarmor-8.0-py2.py3-none-any.whl
Processing ./pyarmor-8.0-py2.py3-none-any.whl
ERROR: Could not find a version that satisfies the requirement pyarmor.cli.core~=1.0 (from pyarmor) (from versions: none)
ERROR: No matching distribution found for pyarmor.cli.core~=1.0
```

**Comment by JiaPai12138** [comment]
Close since it is fixed by 8.0.1
